site-container.yml.sample

```
@@ -531,7 +531,7 @@
       run_once: true
       when: not ceph_status.failed
 
-- hosts: grafana-server
+- hosts: '{{ (groups["grafana-server"] | default(groups["mgrs"]) | default(groups["mons"]))[0] | default(omit) }}'
```

I think we should test this instead:

```suggestion
- hosts: '{{ groups["grafana-server"][0] | default(groups["mgrs"][0]) | default(groups["mons"][0]) | default(omit) }}'
```